lib/html5_f.ml

```diff
@@ -32,11 +32,16 @@
 
 open Html5_types
 
-module Make
+module Make'
```

Comment:
@drup, how about `Wrapped_functions` and `Make_with_wrapped_functions`? Constants is at least vague.